java/spark-1.5.2/backend.scala

```diff
@@ -276,6 +283,13 @@ class Backend {
                     logger.log("is terminating")
                     System.exit(0)
                   }
+
+                  // workers should always terminate but without exceptions
+                  if (isWorker) {
+                    logger.log("is terminating worker")
```

Comment:
`logger.log` concatenates mostly the info you are requesting, but currently, it outputs:

```
17/07/10 17:21:32 INFO sparklyr: Worker (4778) is terminating worker
```

To your point, I think we rather want the following:

```
17/07/10 17:21:32 INFO sparklyr: Worker (4778) is terminating backend
```

Fixed.